pylint/extensions/_check_docs_utils.py

```diff
@@ -377,6 +377,8 @@ def match_param_docs(self):
 
         for match in re.finditer(self.re_param_in_docstring, self.doc):
             name = match.group(2)
+            # Remove espace characters necessary for asterisks
```

Comment:
```suggestion
            # Remove escape characters necessary for asterisks
```

Of course it is now mandatory to the lovely piece of electronic music [Espace/Escape](https://www.youtube.com/watch?v=HaRsnJ1DZmA) by Francis Dhomont.